usage_example.ipynb

```diff
@@ -14,7 +14,8 @@
     "from sequence_classification.results_reader import ResultsReader\n",
     "from sequence_classification.results_writer import ResultsWriter\n",
     "\n",
-    "from sequence_classification.datasets import sentiment, bio_data, troll_detection, valley"
+    "from sequence_classification.datasets import (\n",
+    "    sentiment, bio_data, troll_detection, valley, imdb)"
```

Comment:
A przy okazji może jednak przenieść to do jednej linii bo w notebooku to się spokojnie mieści :D